bin/build.ts

```diff
@@ -40,5 +40,10 @@ main(process.argv)
     process.exit(exitCode)
   })
   .catch(e => {
-    console.error(e)
+    if (e instanceof BuildErrorOutput) {
+      console.error(e.message)
+    } else {
+      console.error(e)
```

Comment:
Can't you always do `console.error(e.message)`? It's a property on `Error`

Reply:
I got it now.

Reply:
If it is unexpected I want the stack trace and extra information